tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1491,6 +1504,26 @@ class OpConverterTest : public ::testing::Test {
     return converter_->quantization_ranges_;
   }
 
+ protected:
+  template <typename T>
+  void AdjustVectorByDims(std::vector<T>& values, size_t num_elements, const string& name, const char *callingFunc) {
```

Comment:
I see code format issue here and other places. Can you use clang format to fix? Other than the format issue, it looks good to me.

Reply:
Done

Reply:
Done